src/background/context-menu.js

```diff
@@ -0,0 +1,73 @@
+import { handleNewRandomAlias } from "./create-alias";
+
+function generateDialogJS(message) {
+  const content = `
+    function showSLDialog() {
+      let slDialog = document.createElement("div");
+      slDialog.style.position = "fixed";
+      slDialog.style.bottom = "0";
+      slDialog.style.right = "0";
+      slDialog.style.margin = "0.7em";
+      slDialog.style.padding = "0.7em";
+      slDialog.style.fontFamily = "Verdana, Arial, Helvetica, sans-serif";
+      slDialog.style.fontSize = "1em";
+      slDialog.style.pointerEvents = "none";
+      slDialog.style.zIndex = "999999";
+      slDialog.style.background = "white";
+      slDialog.style.border = "2px solid #777";
+      slDialog.style.borderRadius = "5px";
+      slDialog.innerText = ${JSON.stringify(message)};
+
+      document.body.appendChild(slDialog);
+
+      setTimeout(function () {
+        document.body.removeChild(slDialog);
+      }, 3000);
+    }
+
+    showSLDialog();
+  `;
+
+  return content;
+}
+
+function generateAliasHandlerJS(tab, res) {
+  const dialogJS = generateDialogJS(
+    res.alias ? res.alias + " copied to clipboard" : "ERROR: " + res.error
+  );
+  const js = `
+    function copyTextToClipboard(text) {
+      if (!text) return;
+      var textArea = document.createElement("textarea");
+      textArea.value = text;
+
+      textArea.style.top = "0";
```

Comment:
could we add `display=none` here to be sure this field isn't visible to user in case of error?

Reply:
it doesn't seems to work with `textArea.style.display = "none"`